iron-resizable-behavior.html

```diff
@@ -102,28 +103,13 @@
      * that should be notified of a resize change.
      */
     stopResizeNotificationsFor: function(target) {
-      index = this._interestedResizables.indexOf(target);
+      var index = this._interestedResizables.indexOf(target);
 
       if (index > -1) {
         this._interestedResizables.splice(index, 1);
       }
     },
 
-    /**
-     * User should implement to introduce filtering when notifying children.
-     * Generally, children that are hidden by the resizable (e.g. non-active
-     * pages) need not be notified during resize, since they will be notified
-     * again when becoming un-hidden.
-     *
-     * Return `true` if the resizable passed as argument should be notified of
-     * resize.
-     *
-     * @method resizerShouldNotify
-     * @param {Element} el
-     */
-    // TODO(cdata): Currently behaviors cannot define "abstract" methods..
```

Comment:
You can get rid of anything or everything except the `TODO` - I would even accept moving the `TODO` :)